conf/httpd.conf.d/httpd.portal

```diff
@@ -201,10 +201,32 @@ $MaxClients = pf::services::manager::httpd::calculate_max_clients(pf::services::
 $StartServers = "4";
 $MinSpareServers = "2";
 
-if( pf::config::isenabled ($PfConfig->{services}{httpd_mod_qos})) {
+if( pf::config::isenabled ($PfConfig->{captive_portal}{httpd_mod_qos})) {
     my $qos = $MaxClients * .7;
     $QS_SrvMaxConnClose = $qos;
-    $QS_SrvMaxConnPerIP = $PfConfig->{services}{httpd_mod_qos_maximum_connections_per_device};
+    $QS_SrvMaxConnPerIP = $PfConfig->{captive_portal}{httpd_mod_qos_maximum_connections_per_device};
+}
+
+if( pf::config::isenabled ($PfConfig->{captive_portal}{httpd_mod_evasive})) {
+    if( pf::config::os_detection() eq 'debian' ) {
```

Comment:
I agree, but can it be loaded but have no effect without any configuration ?
Thinking about mod_qos, which I think, is kind of the same thing no ?